src/io/flutter/inspector/EvalOnDartLibrary.java

```diff
@@ -44,7 +44,7 @@
   CompletableFuture<LibraryRef> libraryRef;
   private final Alarm myRequestsScheduler;
 
-  static final int DEFAULT_REQUEST_TIMEOUT_SECONDS = 5;
+  static final int DEFAULT_REQUEST_TIMEOUT_SECONDS = 10;
```

Comment:
with the other fixes, the timeout doesn't really need to be raised but raising it anyway just to be safe.